assessment.md

```diff
@@ -143,39 +143,96 @@ TODO: add description for each assessment level for this skill.
 
 * Requirements engineering
 
-TODO: add description for each assessment level for this skill.
+Requirements engineering will be evaluated on this scale:
+
+* None = Fails to adds any requirements for the program and shows no knowledge of what the program is doing
+* Inadequate = Adds one requirement for the program
+* Adequate = Adds two requirements for the program
+* Good = Adds three requirements for the program
+* Excellent = Adds more than three requirements for the program and shows knowledge of what the program is doing
```

Comment:
There should be emojis/symbols replacing the words `none`, `inadequate`.. etc. They should be agreed on by all groups. Please make sure to discuss this with others (inside and outside your group) in order to have standard scaling emojis/symbols.

Regarding the content of this skill, how did you come up with these quantities and what's the significance of them. To clarify, requirements engineering as defined by Wikipedia is:

> the process of defining, documenting, and maintaining requirements in the engineering design process

This skill is referring to the ability to document and keep track of project requirements rather than implementing those requirements. I think that the assessment scales should reflect that in a clearer way.

Reply:
We came up with a consensus from a poll saying to have * and =.